packages/react/src/table/TableHeaderSortable.tsx

```diff
@@ -66,8 +58,26 @@ export const TableHeaderSortable = ({
 						},
 					},
 				}}
+				focusRingFor="keyboard"
+				fontWeight="bold"
+				gap={0.5}
+				justifyContent="space-between"
+				onClick={onClick}
+				padding={0.75}
+				width="100%"
 			>
-				{children}
+				<Box
+					as="span"
+					css={{
+						marginLeft:
+							textAlign === 'right' || textAlign === 'center'
+								? 'auto'
+								: undefined,
+						marginRight: textAlign === 'center' ? 'auto' : undefined,
```

Comment:
This won't be right aligned when it wraps but maybe that's ok because heading text won't be long so is unlikely to wrap and the text won't be far from the right if it does?

Reply:
The `td` is using `textAlign: right` so if they both wrap, they'll be different. Again, maybe this is ok?

Reply:
Good pickup. I just pushed a fix